rerun/src/main/java/org/apache/falcon/rerun/handler/RetryHandler.java

```diff
@@ -75,6 +76,9 @@ public void handleRerun(String clusterName, String entityType, String entityName
                         "All retry attempt failed out of configured: "
                                 + attempts + " attempt for entity instance::");
             }
+        } catch (EntityNotRegisteredException ee) {
```

Comment:
I think we should fix this in few other places also. One place i think of is SLAMonitoringHandler for onSuccess  method